### ISSUE TITLE
Enhancement: make asking for download optional

### ISSUE THREAD

**Issue by StefRe** [organicmaps/organicmaps#3737]
When flying in an airplane, I use OM to periodically check my current position. As a rule, I haven't downloaded all the regions I fly over. When OM finds the position it tries to zoom in and asks to download the region in question. I don't want to do this (and in fact can't do it as a rule), so I have to zoom out a bit until the download dialog goes away to see my current position and nearby larger cities on the map. This is not a big issue in principle, but if you have to do this again and again it becomes a bit annoying: after seeing where I am I usually switch off the screen in order to save energy and then after a while when I want to check my position again by tapping on the locating button the app tries do zoom in again and again.  

The proposed enhancement is to add a new option (disabled by default) like "don't ask for download if region of current position is not downloaded" that could be enabled by the user when using OM in the described way. When finding the current position the app would zoom in only to the largest zoom level available without asking to download the current region.  

(there is already an option [`pref_autodownload`](https://github.com/organicmaps/organicmaps/blob/633166ab060dfbc85115ad964e706c8830100550/android/res/xml/prefs_main.xml#L35), so this new option would be in the same category: the existing option controls automatic download and the new option would control the asking for manual download)

What do you think about it?

**Comment by endim8** [comment]
oo, maybe the phone pressure sensor could detect when the altitude is really high & toggle some sort of flight mode that disables the zoom

**Comment by biodranik** [comment]
Do you have a flight mode enabled? Are you connected to WiFi at this moment? Creating a new option for each small case like this one could lead to a cumbersome options dialog that confuses users, let's think if we can solve it smarter and easier.

**Comment by StefRe** [comment]
> Do you have a flight mode enabled? Are you connected to WiFi at this moment?

Yes, I had flight mode enabled and was not connected to WiFi at this moment.

**Comment by biodranik** [comment]
So the proper fix should detect if Airplane mode is enabled (or even better - if there is no internet connection) and not zoom in to avoid showing the Download dialog. There are some edge cases when Airplane mode is on, but the user is connected to WiFi. Actually, we can get the altitude from the GPS and simply detect that the user is on the airplane.

**Comment by endim8** [comment]
it'd be really cute to change the user position marker to an aeroplane when detecting the user is on an aeroplane!

**Comment by biodranik** [comment]
@endim8 it's worth a separate issue )
